Resources/Locale/en-US/_ES/masks/masks.ftl

```diff
@@ -35,6 +32,13 @@ es-mask-infiltrator-desc =  Aid your fellow traitors & complete your objectives
 es-mask-marauder-name = Marauder
 es-mask-marauder-desc = Aid your fellow traitors & complete your objectives to detonate the nuke. You have recieved been trained by the syndicate for loud, brute-force attacks.
 
+es-mask-recruit-name = Recruit
+es-mask-recruit-desc = Youve been recruited to the syndicate for their nefarious plot, help them at all costs.
```

Comment:
I don't love the wording on this but I'm not sure what's the best way to phrase it?

Maybe just something simple like "Aid your fellow traitors & complete your objectives to detonate the nuke. You're an untrained recruit with no equipment, so do your best."

I like all of the traitors having similar descriptions.